example/components/widgets/ExpandedCustomNodeWidget.tsx

```diff
@@ -6,22 +6,24 @@ import {Box, Typography} from "@mui/material";
 import NodeSelection from "./NodeSelection";
 import InputOutputNode from "./InputOutputNode";
 
-export interface CustomNodeWidgetProps {
+const styles = {
+  root: {
+    textColor: {
+      color: '#4579EE',
```

Comment:
@vidhya-metacell should go into the theme instead of "hardcoded" color in the object

Reply:
@zsinnema updated this one

Reply:
@zsinnema if nothing is pending, can this be merged?